DEVELOPING.md

```diff
@@ -54,7 +54,7 @@ The compiled `tla2tools.jar` will be output to `tlatools/org.lamport.tlatools/di
 Build & Test Toolbox IDE
 ------------------------
 Install the following dependencies to your path:
- * [Java Development Kit](https://adoptium.net/) version 11+
+ * [Java Development Kit](https://adoptium.net/) version 11 (newer versions will likely cause build failures)
```

Comment:
Include a link to https://github.com/tlaplus/tlaplus/issues/1162#issuecomment-2737943830